markdown/usage/installation.md

```diff
@@ -31,125 +17,60 @@ java -version
-  }
-}
-
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-```
+We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
 
-This uses a number of sensible defaults for process requirements and is suitable for running
-on a simple (if powerful!) local server.
+Updating nextflow is as simple as running `nextflow self-update`.
 
-Be warned of two important points about this default configuration:
+Once installed you will probably need to configure Nextflow to run on your system. For instructions, see [_Nextflow configuration_](configuration).
 
-1. The default profile uses the `local` executor
-    * All jobs are run in the login session. If you're using a simple server, this may be fine. If you're using a compute cluster, this is bad as all jobs will run on the head node.
-    * See the [nextflow docs](https://www.nextflow.io/docs/latest/executor.html) for information about running with other hardware backends. Most job scheduler systems are natively supported.
-2. Nextflow will expect all software to be installed and available on the `PATH`
-    * It's expected to use an additional config profile for docker, singularity or conda support. See below.
 
-### Docker
-First, install docker on your system: [Docker Installation Instructions](https://docs.docker.com/engine/installation/)
+## Pipeline software
 
-Then, running the pipeline with the option `-profile docker` tells Nextflow to enable Docker for this run.
-An image containing all of the software requirements will be automatically fetched and used from dockerhub
-([https://hub.docker.com/r/nfcore/YOUR_PIPELINE](https://hub.docker.com/r/nfcore/YOUR_PIPELINE)).
+An analysis pipeline chains the execution of multiple tools together.
+Historically, all tools would have to be manually installed, often a source of great frustration and a key step where reproducibility between analyses is lost.
+nf-core pipelines utilise the built-in support for software packaging that Nextflow offers: all can work with Docker and Singularity, and most pipelines also have support for Conda.
 
-### Singularity
-If you're not able to use Docker then [Singularity](http://singularity.lbl.gov/) is a great alternative.
-The process is very similar: running the pipeline with the option `-profile singularity` tells Nextflow to enable singularity for this run. An image containing all of the software requirements will be automatically fetched and used from singularity hub.
+* [Docker](https://docs.docker.com/install/)
+  * Typically used locally / on single-user servers and the cloud.
+  * Analysis runs in a _container_, which behaves like an isolated operating system
+  * Previously required system root access, though a "rootless mode" is available since late 2019
+  * Best option for ultimate reproducibility
```

Comment:
How does it compare with singularity?
You're saying here that Docker is the best for that, and in the Conda part it seems that Docker and Singularity are at the same level

Reply:
Yeah. I agree. Is it any better than `Singularity` for reproducibility?

Reply:
Probably not..

Reply:
Removed this, and made the conda lower reproducibility argument a bit stronger / clearer.